1.9/administration/installing/custom/cli.md

```diff
@@ -133,7 +133,7 @@ The DC/OS installation creates these folders:
 
     In this step you create a YAML configuration file that is customized for your environment. DC/OS uses this configuration file during installation to generate your cluster installation files.
 
-    You can use this template to get started. This template specifies 3 masters, 5 [private](/docs/1.9/overview/concepts/#private) agents, 1 [public](/docs/1.9/overview/concepts/#public) agent, and SSH configuration specified. If your servers are installed with a domain name in your `/etc/resolv.conf`, you should add `dns_search` to your `config.yaml` file. For parameters descriptions and configuration examples, see the [documentation][6].
+    You can use this template to get started. This template specifies 3 masters, 5 [private](/docs/1.9/overview/concepts/#private) agents, 1 [public](/docs/1.9/overview/concepts/#public) agent, a custom proxy, and SSH configuration specified. If your servers are installed with a domain name in your `/etc/resolv.conf`, you should add `dns_search` to your `config.yaml` file. For parameters descriptions and configuration examples, see the [documentation][6].
```

Comment:
@jsancio Local universe doc is here: https://docs.mesosphere.com/1.8/administration/installing/custom/deploying-a-local-dcos-universe/